interfaces/apparmor/spec_test.go

```diff
@@ -615,3 +615,55 @@ func (s *specSuite) TestSetSuppressPycacheDeny(c *C) {
 	s.spec.SetSuppressPycacheDeny()
 	c.Assert(s.spec.SuppressPycacheDeny(), Equals, true)
 }
+
+func contains(slice []string, needle string) bool {
```

Comment:
This is both `slices.Contains` in modern go but most importantly `strutil.ListContains` in snapd.

Reply:
For tests we also have `testutil.Contains`, which was my very fist contribution to snapd :)

Reply:
Yes, I tried `slices.Contains` first and `it worked on my machine(tm)`, but when I uploaded to github, the tests failed. That's why I wrote it. But you are right about changing it with `testutil.Contains`.

Reply:
Done.